src/main.hs

```diff
@@ -9,4 +9,4 @@ main = getArgs >>= deicide
     "compress"   -> print $ compress $ stringToInteger $ a !! 1
     "decompress" -> putStrLn $ integerToString $
                     decompress (read $ a !! 1, read $ a !! 2)
-    _            -> error "An invalId command is entered.";
```

Comment:
The man who is responsible for this typo should receive a nice slap.

Aw, dang.